lambdas/user_access_patterns/lambda_for_createUser/package.json

```diff
@@ -16,5 +16,8 @@
   "bugs": {
     "url": "https://github.com/code-gambit/VT-lambdas/issues"
   },
-  "homepage": "https://github.com/code-gambit/VT-lambdas#readme"
+  "homepage": "https://github.com/code-gambit/VT-lambdas#readme",
+  "dependencies": {
+    "aws-sdk": "^2.883.0"
```

Comment:
we can avoid this also, incase you are not using any dependency apart from `aws-sdk` then no need to add this here, simply remove the entire `"dependencies"` object from package.json and AWS will automatically add it.

Reply:
For testing we need it. And if someone decides to build the project locally, it would be helpful for them.

Reply:
mention this in building guidelines, and push only the production code not testing code.

Reply:
From testing I mean that if you wanna check if lambda is working fine from local setup, then its convinient or else for every lambda you have to write npm install aws-sdk --save. I think its better to have every module which is required in index.js to be in package.json file.

Reply:
fine